src/pages/collections/[slug].tsx

```diff
@@ -33,6 +33,8 @@ type CollectionItemListProps = {
   noPagination?: boolean;
 };
 
+const TextBodyL = (props) => <Text {...props} variant="body-l" />;
```

Comment:
I think props might be `any` here. I think we could consider whether we want to give this a type. For instance with `React.HTMLAttributes<HTMLParagraphElement>`, which should be close to the props a `p` element can receive.

The result is a bit uglier declaration due to slight differences between the attributes of a `p` element and the props of a `Text`.

```tsx
const TextBodyL = ({
  children,
  ...props
}: React.HTMLAttributes<HTMLParagraphElement>) =>
  children ? (
    <Text {...props} variant="body-l">
      {children}
    </Text>
  ) : null;
```

Maybe we could align these APIs by making children optional in the `Text` component?

Your call.